share/sensitivity-analysis/dakota_test.in

```diff
@@ -0,0 +1,25 @@
+# Demonstrate Cyclus-Dakota Coupling
+# One-at-a-time sensitivity analysis of power capacity for a reactor facility. 
+
+environment, 
+	tabular_data 
+		tabular_data_file = 'dakota_test.dat'
```

Comment:
Is this file created, or is it input?  If the latter, it's missing from the PR....

Reply:
It is a created file with the results from the simulation